extensions/elation/actions/createCareGap/createCareGap.ts

```diff
@@ -0,0 +1,47 @@
+import { Category, type Action } from '@awell-health/extensions-core'
+import { makeAPIClient } from '../../client'
+import { type settings } from '../../settings'
+import { dataPoints, fields, FieldsValidationSchema } from './config'
+
+export const createCareGap: Action<
+  typeof fields,
+  typeof settings,
+  keyof typeof dataPoints
+> = {
+  key: 'createCareGap',
+  category: Category.EHR_INTEGRATIONS,
+  title: 'Create care gap',
+  description: 'Create a care gap in Elation.',
+  fields,
+  previewable: true,
+  dataPoints,
+  onActivityCreated: async (payload, onComplete, onError): Promise<void> => {    
+    const { patient_id, practice_id, created_date, status, detail, quality_program, definition_id } = FieldsValidationSchema.parse(payload.fields)
```

Comment:
Just checking to see if you have set up your formatter as I would have maybe expected this to be on a new line given the length:
 
```
const { 
   patient_id, 
   practice_id, 
   created_date, 
   status, detail, 
   quality_program, 
   definition_id 
}
```

Reply:
The current setup of this repo is really poor. I intend to come back to this during one of the next cooldowns to set up proper hooks to take care of that. In the meantime I won't try to configure my IDE as that would be a bit of a waste of time.